app/views/devise/sessions/new.html.erb

```diff
@@ -34,14 +34,14 @@
       ) %>
   <%= f.input :request_id, as: :hidden, input_html: { value: @request_id } %>
   <div class='margin-bottom-4'>
-    <%= f.submit t('links.next'), full_width: true, big: false, wide: false %>
+    <%= f.submit t('links.next'), full_width: true, big: true, wide: false %>
```

Comment:
`big` is [the default for the underlying submit component](https://github.com/18F/identity-idp/blob/688a0d104f23296459a3316bb5282e691d24c360/app/components/submit_button_component.rb#L2), so it was only necessary to specify when explicitly making it smaller. Now we could choose to leave it out.

```suggestion
    <%= f.submit t('links.next'), full_width: true, wide: false %>
```